pom.xml

```diff
@@ -64,33 +62,21 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>org.jfree</groupId>
             <artifactId>jfreechart</artifactId>
             <version>1.5.3</version>
         </dependency>
-        <dependency>
```

Comment:
Jaký je důvod vymazaní lomboku? Prosím objasníš to? Děkuji

Reply:
podla toho co si oznacila nevidim ze by som mazal lombok, ak je to tak asi to bola duplicita, pretoze v tejto branch lombok je, staci pozriet pom.xml